integrations/amazon_bedrock/tests/test_chat_generator.py

```diff
@@ -71,12 +72,16 @@ def test_from_dict(mock_boto3_session):
                 "generation_kwargs": {"temperature": 0.7},
                 "streaming_callback": "haystack.components.generators.utils.print_streaming_chunk",
                 "truncate": True,
+                "boto3_config": {
```

Comment:
did you test that the `from_dict` can handle a serialized object without the newly added `boto3_config`? we need to be sure this change is still compatible with older versions

Reply:
Done. I parametrized the tests for the boto3_cofig value.